src/components/ProjectMemberCard/ProjectMemberCard.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  Card,
+  CardActions,
+  CardContent,
+  makeStyles,
+  Typography,
+} from '@material-ui/core';
+import { Skeleton } from '@material-ui/lab';
+import { FC } from 'react';
+import * as React from 'react';
+import { displayRole } from '../../api/displayRole';
+import { displayUserName } from '../../api/displayUserName';
+import { Avatar } from '../Avatar';
+import { useDateFormatter } from '../Formatters';
+import { ButtonLink } from '../Routing';
+import { ProjectMemberFragment } from './ProjectMember.generated';
+
+const useStyles = makeStyles(({ palette, spacing }) => ({
+  cardContent: {
+    display: 'flex',
+  },
+  cardActions: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    borderTop: `0.5px solid ${palette.divider}`,
+    padding: spacing(1, 2, 1, 1),
+  },
+  avatar: {
+    width: spacing(7),
+    height: spacing(7),
+    marginRight: spacing(2),
+  },
+  memberInfo: {
+    flexGrow: 1,
+  },
+}));
+
+export interface ProjectMemberCardProps {
+  imageSource?: string;
```

Comment:
Same for this as coming from user. But we won't have it for initial release. Need to fallback to avatarLetters